docs/source/usersguide/input.rst

```diff
@@ -1116,6 +1116,15 @@ Each ``material`` element can have the following attributes or sub-elements:
     .. note:: If one nuclide is specified in atom percent, all others must also
               be given in atom percent. The same applies for weight percentages.
 
+    An optional attribute/sub-element for each nuclide is ``scattering``. This
+    attribute may be set to "ace" to use the scattering laws specified in the
+    ACE files (default). Alternatively, when set to "lab", the ACE scattering
```

Comment:
Also, if you have a particular suggestion of how you would rephrase this description without referencing ACE then I will use it; otherwise, I'm unsure of what would work well here.